src/module/PluginsModule.ts

```diff
@@ -0,0 +1,34 @@
+import { IInstalledPlugin } from '../models/Plugins';
+import { Unauthorized } from '../response/errors';
+import { Success } from '../response/success';
+
+/**
+ * LINKURIOUS CONFIDENTIAL
+ * Copyright Linkurious SAS 2012 - 2019
+ *
+ * - Created on 2019-09-17.
+ */
+import {Module} from './Module';
+
+export class PluginsModule extends Module {
+
+  /**
+   * Get the list of installed plugins.
+   */
+  public getPlugins(): Promise<Success<IInstalledPlugin[]> | Unauthorized> {
```

Comment:
Let's stick to response interfaces:
Promise<Success<IGetPluginsResponse> | ...

then you can define :
- type IGetPluginsResponse = IInstalledPlugin[]